src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -104,7 +105,8 @@ public EventTypeService(final EventTypeRepository eventTypeRepository,
             eventType.setTopic(topicName);
             eventTypeRepository.saveEventType(eventType);
             return Result.ok();
-        } catch (final InvalidEventTypeException | NoSuchPartitionStrategyException e) {
+        } catch (final InvalidEventTypeException | NoSuchPartitionStrategyException |
+                DuplicatedEventTypeNameException e) {
```

Comment:
I addressed your comment in https://github.com/zalando/nakadi/pull/615/commits/fb1025e1010868dbd93791071b24b73c3f0bdac3